tools/cockpit.spec

```diff
@@ -523,6 +523,22 @@ getent group cockpit-wsinstance >/dev/null || groupadd -r cockpit-wsinstance
 getent passwd cockpit-wsinstance >/dev/null || useradd -r -g cockpit-wsinstance -d /nonexisting -s /sbin/nologin -c "User for cockpit-ws instances" cockpit-wsinstance
 
 %post ws
+if firewall-cmd --state; then
+    firewall-cmd --zone public --add-service cockpit
```

Comment:
Please wrap these in a `for` loop and define a constant for the zones, to avoid repeating them three times.

Is there some way of enumerating the zones? It could be that some of them don't exist, right?

Is cockpit currently enabled in all of these? Adding it to `public` is probably the most dangerous one, but of course we should stay compatible to what happens now.

Reply:
so we could also just add it to/ remove it from whatever the default zone would be.

`public` would be the most dangerous one but also the most useful one :/